layouts/json/single.html

```diff
@@ -2,6 +2,8 @@
 {"Posts": [
   {{ range $k, $post := (sort $pages.Pages "Date" "desc") }}
     {{if $k}},{{end}}
-    {{ dict "title" $post.LinkTitle "description" $post.Description "link" $post.Permalink "date" ($post.Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML) | jsonify }}
+
```

Comment:
I needed to add air to the code in some place. Otherwise, it would be a loop with three lines of WTF.